From 88506634f7478104eeab647819903ea6292a5d2d Mon Sep 17 00:00:00 2001
From: Scott Lobdell <slobdell@google.com>
Date: Mon, 28 Jan 2019 15:44:33 -0800
Subject: [PATCH 32/37] Fix build errors

 - include <sys/uio.h> in order to resolve use of undeclared identifier
 'writev'
  - implement abstract methods for QC value-adds on BaseWifiService
  - update files to match in specs in api/current.txt and api/system-current.txt
  - media/jni/Android.bp needed a static dependency moved to shared
  object dependency

Bug: 123699636
Change-Id: I30f97e24e83274016e720548fba412da0f35c98f
---
 core/java/android/bluetooth/BluetoothHeadset.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/java/android/bluetooth/BluetoothHeadset.java b/core/java/android/bluetooth/BluetoothHeadset.java
index 5d5a3d2afe1..3625ec22b7f 100644
--- a/core/java/android/bluetooth/BluetoothHeadset.java
+++ b/core/java/android/bluetooth/BluetoothHeadset.java
@@ -452,7 +452,7 @@ public final class BluetoothHeadset implements BluetoothProfile {
         doUnbind();
     }
 
-    protected void finalize() {
+    protected void finalize() throws Throwable {
         close();
     }
 
-- 
2.28.0

